resources/migrate-to-swift.md

```diff
@@ -53,13 +53,45 @@ Once the previous command is complete, run `pod install` or `pod update` to upda
 
 ### Swift Package Manager
 
-You can now also use Swift Package Manager (SPM) to manage your Adobe Experience Platform Mobile SDK dependencies. To add the AEP SDK Packages to your application, from the Xcode 13 menu select:
+#### Swift Package Collection
+
+In Swift 5.5, the Swift Package Manager (SPM) adds support for [package collections](https://www.swift.org/blog/package-collections). You can configure package collection in Xcode 13 for easy installation of AEP SDKs. 
+
+Swift Package Collection for AEP SDKs is available at 
+```
+  https://opensource.adobe.com/aepsdk-core-ios/swift/packages/aep.json
+```
+
+To add Swift Package Collection, from the Xcode 13 menu select: 
+
+```
+File > Add Packages...
+```
+
+Now, click on the plus sign on the bottom left and click "Add Swift Package Collection".
+
+![](../.gitbook/assets/spm-add-package-collection.png)
+
+Next, enter the package collection url and click "Load". After the package collection has loaded, click "Add Collection" to add the collection.
+
+![](../.gitbook/assets/spm-add-package-collection-load.png)
+
+You should now see the added package collection on the left pane. Once selected, you will see all of the packages included in the collection listed.
+
+![](../.gitbook/assets/spm-package-collection-aep.png)
+
+
+#### Installing AEP SDKs using SPM 
+
+To add the AEP SDK Packages to your application, from the Xcode 13 menu select:
 
 ```
 File > Add Packages...
 ```
 
-Enter the Package URL for the AEP SDK repositories: 
+If you have configured package collection as mentioned above, simply select each package you would like to add to your project and click "Add Package" on the bottom right.
```

Comment:
"... as mentioned above, select each package you would like..."

(We generally remove stuff like "simply select" since it's a bit too fluffy.)